code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -408,3 +408,27 @@ Contains:
 	armor = list(melee = 60, bullet = 40, laser = 40, energy = 50, bomb = 100, bio = 100, rad = 100, fire = 100, acid = 100)
 	strip_delay = 130
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+

```

Comment:
Please move all of this into the NSV13 clothing files as we try not to cause potential conflicts from upstream, ie /nsv13/code/modules/custom_clothes.dm